hr_employee_phone_extension/readme/CONTRIBUTORS.rst

```diff
@@ -7,3 +7,4 @@
 * Anand Kansagra <kansagraanand@hotmail.com>
 * Saran Lim. <saranl@ecosoft.co.th>
 * aphon61bank <aphon61bank@gmail.com>
+* Carolina Fernandez <carolina.fernandez@tecnativa.com>
```

Comment:
Don't put your mail. You can use the two lines form

Reply:
Done!